aws/ami/windows/windows.pkr.hcl

```diff
@@ -40,6 +40,20 @@ source "amazon-ebs" "windows_ebs_builder" {
 build {
   sources = ["source.amazon-ebs.windows_ebs_builder"]
 
+  # Install conda, it needs to be installed under SYSTEM to avoid this broken
+  # installation https://github.com/ContinuumIO/anaconda-issues/issues/11799.
+  # Also this needs to come after all the tools are installed to avoid error
+  # CondaHTTPError: HTTP 000 CONNECTION FAILED when connecting to conda (?)
```

Comment:
Is this part of the comment no longer accurate, where it says it needs to be installed after all the other tools?

The issue implies that the installation actually fails but still appears to have succeeded. How would we test this?

Reply:
Right looks like its not accurate anymore. let me fix it